### ISSUE TITLE
GM security

### ISSUE THREAD

**Issue by jacobp100** [uber/image-diff#37]
Image-diff requires an insecure version of `gm`.

```
Command Injection in gm.compare function
View advisory details
Discovered in a nested dependency:image-diff@1.5.0 gm@1.13.3
```

See https://nodesecurity.io/advisories/54

**Comment by twolfson** [comment]
Fortunately we are using our own `compare` invocation with `buffered-spawn` so we don't have that vulnerability.

https://github.com/uber/image-diff/blob/1.5.0/lib/image-diff.js#L70-L96

However, I agree that an upgrade would be best to prevent potential future regressions. Would you be interested in opening a PR?

**Comment by twolfson** [comment]
@jacobp100 This should be resolved now that #38 has been landed and released in `1.5.1`. Can you please close this issue? (I no longer have write access on the repo despite being the author =/)
